src/Service/Models/AccountMetaInformation.cs

```diff
@@ -24,4 +25,5 @@ public string AdminEmailsSerialized
     public DateTime? DeleteAt { get; set; }
 
     public AppFeature Features { get; set; }
+    public IEnumerable<ApplicationEvent> Events { get; set; }
```

Comment:
@jrmccannon are we 100% that thie doesn't get loaded up when we load AccountMetaInformation from the db?

If they get loaded by default it will severely impact perfromance. Could we test?

Reply:
Didn't notice this, but we should avoid using `IEnumerable<T>` properties because they might map to some lazily evaluated collection (e.g. from EF) and that might result in some deferred exceptions. A better option would be `IReadOnlyCollection<T>` or `IReadOnlyList<T>`.

Reply:
@justindbaur Do you know if there is some way to "guarantee" we're not loading this?

Reply:
`AccountMetadataInformation` has a one-to-many relationship with `ApplicationEvent`? Then you get the guarantee that EF won't load that by just NOT doing a `Include()` or `ThenInclude` on queries that would request it. Some people will make this property virtual to sort of signify that it's another table that isn't always there but that is purely optional. 

A lot of docs will make the property be an `ICollection<T>` https://learn.microsoft.com/en-us/ef/core/modeling/relationships/one-to-many but `IReadOnlyCollection` should be fine.

Reply:
Thanks @justindbaur.
Let's add Virtual and we're good to go @jrmccannon

Reply:
Done